src/app/portlets/dot-apps/dot-apps-import-export-dialog/dot-apps-import-export-dialog.component.ts

```diff
@@ -0,0 +1,194 @@
+import {
```

Comment:
I feel unified this component make it way too complex. For example you are setting `this.dialogActions` 3 times in the lifetime for the component.